docs/USAGE.md

```diff
@@ -0,0 +1,284 @@
+| `OTEL_TRACE_DOMAIN_NEUTRAL_INSTRUMENTATION` |  Sets whether to intercept method calls when the caller method is inside a domain-neutral assembly. This is recommended when instrumenting IIS applications. | `false` |
+| `OTEL_PROFILER_PROCESSES` | Sets the filename of executables the profiler can attach to. If not defined (default), the profiler will attach to any process. Supports multiple values separated with semi-colons, for example: `MyApp.exe;dotnet.exe` |  |
+| `OTEL_PROFILER_EXCLUDE_PROCESSES` | Sets the filename of executables the profiler cannot attach to. If not defined (default), the profiler will attach to any process. Supports multiple values separated with semi-colons, for example: `MyApp.exe;dotnet.exe` |  |
+
+
+### Linux
+
+After downloading the library, install the CLR Profiler and its components
+via your system's package manager.
+
+1. Download the latest release of the library.
+2. Install the CLR Profiler and its components with your system's package
+manager:
+    ```bash
+    # Use dpkg:
+    $ dpkg -i otel-dotnet-autoinstrumentation.deb
+
+    # Use rpm:
+    $ rpm -ivh otel-dotnet-autoinstrumentation.rpm
+
+    # Install directly from the release bundle:
+    $ tar -xf otel-dotnet-autoinstrumentation.tar.gz -C /
+
+    # Install directly from the release bundle for musl-using systems (Alpine Linux):
+    $ tar -xf otel-dotnet-autoinstrumentation-musl.tar.gz -C /
+    ```
+1. Configure the required environment variables:
+    ```bash
+    $ source /opt/otel-dotnet-autoinstrumentation/defaults.env
+    ```
+2. Set the service name:
+    ```bash
+    $ export OTEL_SERVICE='MyCoreService'
+    ```
+3. Set Zipkin exporter:
+    ```bash
+    $ export OTEL_EXPORTER='Zipkin'
+    ```
+4. Set OpenTelemetry conventions:
+    ```bash
+    $ export OTEL_CONVENTION='OpenTelemetry'
+    ```
+5. Set the endpoint, e.g. OpenTelemetry Collector:
+    ```bash
+    $ export OTEL_TRACE_AGENT_URL='http://<YourCollector>:9080/v1/trace'
+    ```
+6. Optionally, enable trace injection in logs:
+    ```bash
+    $ export OTEL_LOGS_INJECTION=true
+    ```
+7. Optionally, create the default logging directory:
+    ```bash
+    $ source /opt/otel-dotnet-autoinstrumentation/createLogPath.sh
+    ```
+8. Run your application:
+    ```bash
+    $ dotnet run
+    ```
+
+### Windows
+
+**Warning**: Pay close attention to the scope of environment variables. Ensure they
+are properly set prior to launching the targeted process. The following steps set the
+environment variables at the machine level, with the exception of the variables used
+for finer control of which processes will be instrumented, which are set in the current
+command session.
+
+1. Install the CLR Profiler using an installer file (`.msi` file) from the latest release.
+Choose the installer (x64 or x86) according to the architecture of the application
+you're instrumenting.
```

Comment:
```suggestion
Choose the installer (x64 or x86) according to the architecture of the operating
system where it will be running.
```

Reply:
This is because the x64 MSI will install both the 32-bit and 64-bit profiler onto the machine. The x86 MSI will only install the 32-bit profiler onto the machine, since it can only run 32-bit processes.